### PR TITLE
[CLA] CLA Signature for chowbryan

### PR DESCRIPTION
**Pull request by chowbryan**
CLA signature

### CODE REVIEW

File: doc/cla/individual/chowbryan [hunk 1]
```diff
@@ -0,0 +1,13 @@
+I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr
```
Comment (mart-e) [inline doc/cla/individual/chowbryan:1]: this line is not necessary

File: doc/cla/individual/chowbryan [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+United States, 2/3/2022
```
Comment (mart-e) [inline doc/cla/individual/chowbryan:3]: could you use the `year-month-day` format? the american date format is unclear (3rd of Feb or 2nd of March)

**Review by mart-e — COMMENTED** [review]
Hello,

Thank you for the CLA signature but it seems there is still an issue.
Your file must be named `chowbryan.md` and not `chowbryan` to be detected by our [CLA ci](https://github.com/odoo/runbot/blob/13.0/runbot_cla/build_config.py#L20).

Thanks

**Comment by mart-e** [comment]
Hey actually, we are collegues and you work at odoo ! :D
So no need for a CLA signature, just make your commits with your `@odoo.com` email address and push your code in the [odoo-dev fork](https://github.com/odoo-dev/odoo/).

**Comment by chowbryan** [comment]
> Hey actually, we are collegues and you work at odoo ! :D So no need for a CLA signature, just make your commits with your `@odoo.com` email address and push your code in the [odoo-dev fork](https://github.com/odoo-dev/odoo/).

Hey! Thanks for reviewing and sorry for the messy CLA. If you don't mind, I'll submit a new one since I want to use this account to start building my 

> Hey actually, we are collegues and you work at odoo ! :D So no need for a CLA signature, just make your commits with your `@odoo.com` email address and push your code in the [odoo-dev fork](https://github.com/odoo-dev/odoo/).

Updated my email. Thank you! (sorry for the messy CLA also)

**Pull request closed without merging**